src/mjbots_hardware_interface.cpp

```diff
@@ -118,9 +120,18 @@ void MjbotsHardwareInterface::ProcessReply() {
 
 void MjbotsHardwareInterface::SendCommand() {
   cycle_count_++;
-  
-  for (int servo=0; servo < num_joints_;servo++) {// TODO Move to a seperate update method (allow non-ff torque commands)?
-    commands_[servo].position.feedforward_torque = joints[servo]->get_servo_torque();
+
+  if (!dry_run_) {
+    for (int servo = 0; servo < num_joints_; servo++) {// TODO Move to a seperate update method (allow non-ff torque commands)?
+      commands_[servo].position.feedforward_torque = joints[servo]->get_servo_torque();
+    }
+  } else {
+    std::fprintf(stdout, "Torques: ");
+    for (int servo = 0; servo < num_joints_; servo++) {
+      commands_[servo].position.feedforward_torque = 0.0;
+      std::fprintf(stdout, "% 8.2f, ", joints[servo]->get_servo_torque());
```

Comment:
Why are we using print and not the logging macros?

Reply:
I think because it is a vector

Reply:
In this case laziness.  I wanted to print a full each element of the `std::vector` on a single line, and we do not currently have a macro for printing vectors of values.  Might be a good addition though.

Reply:
Also, I think that I am going to add a print_torques bool, because as expected, printing is a little aggressive.

Reply:
@ShaneRozenLevy Thoughts or revert?

Reply:
Probably fine. I don't have super strong feelings about this.

Reply:
Fixed in d600e77fe0a5fce1e9499f2b68c09096d39340f3